pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "src"
+version = "1.0.0"
+description = "Search frontend to MongoDB stored scraped Parler data."
+authors = ["Seattle Tech Bloc <techblocsea@protonmail.com>"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+aioredis = "^1.3.1"
+python = "^3.9"
+Quart = "^0.15.1"
+Quart-Motor = "^2.4.4"
+Quart-Rate-Limiter = "^0.5.0"
+python-dotenv = "^0.18.0"
+
+[tool.poetry.dev-dependencies]
+pytest = "^6.2.4"
+black = "^21.6b0"
+flake8 = "^3.9.2"
+mypy = "^0.910"
+pre-commit = "^2.13.0"
+pytest-asyncio = "^0.15.1"
+pytest-pythonpath = "^0.7.3"
```

Comment:
Is this meant to replace `requirements.txt`?

Reply:
Oh yeah it would replace it if we went with this, we could delete requirements.txt and would want to update the README and everything.